capstone/src/test/javascript/graphsSpec.js

```diff
@@ -0,0 +1,26 @@
+const google = {};
+google.visualization = {};
+google.visualization.DataTable = class {
+  /**
+   * Dummy constructor for mock DataTable class.
+   */
+  constructor() {}
+};
+
+describe('Data Table Construction', () => {
+  it('Will add correct columns/rows to DataTable from JSON', () => {
+    let expectedData = {};
+    let actualData = {};
+    const mockTuitionInfo = {};
+    window.initTuitionData = () => {
+      // TODO(ihsan314): Initialize this when format of
+      // JSON object is well-defined
+      expectedData = new google.visualization.DataTable();
+      actualData = new google.visualization.DataTable();
+      console.log(expectedData);
+      console.log(actualData);
+      populateDataTable(actualData, mockTuitionInfo);
```

Comment:
I don't think this block of code is ever invoked. 

And I think this should test `deserializeTuitionData` instead of populateDataTable. That way we can mock our response and make sure it converts correctly...

Reply:
Since I'm mocking it anyway I don't need the `initTuitionData` callback so I'll remove it. Additionally I'll change the test to test `deserializeTuitionData`.